sample/src/main/java/io/fotoapparat/sample/MainActivity.java

```diff
@@ -79,17 +110,10 @@ protected void onCreate(Bundle savedInstanceState) {
                         fileLogger(this)
                 ))
                 .build();
-
-        cameraView.setOnClickListener(new View.OnClickListener() {
-            @Override
-            public void onClick(View v) {
-                takePicture();
-            }
-        });
     }
 
     private void takePicture() {
-        PhotoResult photoResult = fotoapparat.takePicture();
+        PhotoResult photoResult = fotoapparatSwitcher.getCurrentFotoapparat().takePicture();
```

Comment:
Yeah, I also thought about it, but it would explode the API of Switcher and we'll always have to support it. I suggest we proceed with current API, and then after 1.1, we'll see how big API of Fotoapparat will become. If it won't grow noticeably, then we'll do as you say.